scripts/test_unit.sh

```diff
@@ -6,6 +6,8 @@ if [ -z ${GO_ENV} ]; then
     export GO_ENV="test"
 fi
 
+export GO111MODULE=on
```

Comment:
Why do we need this?

Reply:
I added it in PR description but will copy/paste it here 

> Also added export GO111MODULE=on to scripts/test_unit.sh because when working from GOPATH and when this env is not set I recieved error about mod flag. Is that ok or I should remove setter for that env?

Is there a reason why we don't added it here? Because we do similar thing in test_e2e.sh

Reply:
Oh got it! thanks! I realized I do not have GOPATH set so never landed into this bug!